specs/core/0_beacon-chain.md

```diff
@@ -1088,28 +1075,75 @@ def get_domain(fork: Fork,
     ) * 2**32 + domain_type
 ```
 
+#### `get_bitfield_bit`
+
+```python
+def get_bitfield_bit(bitfield: bytes, i: int) -> int:
```

Comment:
What do you think about renaming `i` to `index`?

Reply:
Suggestion:
We return `bool` in Python codebase since the use cases are usually checking if the bit is 0 or not.
```python
def has_voted(bitfield: Bitfield, index: int) -> bool:
    return bool(bitfield[index // 8] & (128 >> (index % 8)))
```

Reply:
I'm trying to reserve `index` for validator indices.